pkg/types/nutanix/validation/platform_test.go

```diff
@@ -206,7 +206,7 @@ func TestValidatePlatform(t *testing.T) {
 				tc.config = installConfig().build()
 				tc.config.Nutanix = tc.platform
 			}
-			err := ValidatePlatform(tc.platform, field.NewPath("test-path"), tc.config).ToAggregate()
+			err := ValidatePlatform(tc.platform, field.NewPath("test-path"), tc.config, false).ToAggregate()
```

Comment:
This is not agent based installer code but IPI so you should not required making changes in this code path

Reply:
Cc @sadasu

Reply:
This change is because a new parameter "usingAgentMethod bool" is added to the func ValidatePlatform() in [pkg/types/validation/installconfig.go]. Because the func validateNutanixPlatform() in pkg/asset/agent/installconfig.go also calls this func for validation.

Reply:
@pawanpinjarkar seems to be fine. They have a common validation method with an input flag that indicates if install method is ABI. If so, a smaller set of validations are run.